zenoh/src/queryable.rs

```diff
@@ -150,6 +160,36 @@ pub struct ReplyBuilder<'a> {
     result: Result<Sample, Value>,
 }
 
+impl<'a> ReplyBuilder<'a> {
+    #[zenoh_macros::unstable]
+    pub fn attachment(&self) -> Option<&Attachment> {
```

Comment:
Similar comment as in the other builders... What's the interest of attachment and attachment_mut in the PutBuilder? Shouldn't with_attachment be enough? It's a builder, so I would expect to be additive for the attachment...

Reply:
Agreed, and this spot is especially painful